source/lba1/compile.rst

```diff
@@ -1,2 +1,143 @@
 Compile
-=======
\ No newline at end of file
+=======
+
+Prerequisites
+-------------
+
+-  `DOSBox <https://www.dosbox.com/>`__ - DOS emulator which we will
+   use to compile the game inside.
+-  `4DOS <https://www.4dos.info/v4dos.htm#751>`__ - Command line
+   interpreter, which supports the ``copy`` command with binary
+   inputs and output.
+-  Watcom 10 compiler - For compiling C sources and running MAKEFILEs
+-  MASM (Microsoft Macro Assembler) 6.0 - For compiling ASM sources
+
+Getting prerequisites and sources
+---------------------------------
+
+DOSBox and 4DOS are freely available. For getting Watcom 10 and MASM
+6.0, you need to search the internet. Note that we did not manage to
+build the game with Open Watcom. Also, for some reason the MASM version
+6.11 compiler did run very slowly in the DOSBox, so it was basically
+unusable. We had to use the version 6.0.
+
+All directories and files will placed in the ``~/lba-hacking`` directory
```

Comment:
Perhaps instead of "-hacking" it can use a more generic "-compile" word instead.